xtc/src/stats.rs

```diff
@@ -4,9 +4,23 @@ use ic_cdk::*;
 use ic_cdk_macros::*;
 use serde::Deserialize;
 
+#[derive(Deserialize, CandidType, Clone)]
+pub struct StatsDataV0 {
+    supply: Nat,
+    history_events: u64,
+    balance: u64,
+    // Usage statistics
+    transfers_count: u64,
+    mints_count: u64,
+    burns_count: u64,
+    proxy_calls_count: u64,
+    canisters_created_count: u64,
+}
+
 #[derive(Deserialize, CandidType, Clone)]
 pub struct StatsData {
     supply: Nat,
+    fee: Nat,
```

Comment:
should we change the candid file too? If I am not mistaken adding a new field is a compatible change.

Reply:
yes we should change the candid